src/elo/constants/eloConstants.ts

```diff
@@ -1,15 +1,15 @@
 const eloConstants = {
-    // ELO System Constants
-    DEFAULTRATING: 1500,
-    DEFAULTRD: 350,
-    DEFAULTVOL: 0.06,
-    PROVISIONGAMES : 20,
-    BRONZEBASE : 1300,
-    SILVERBASE : 1400,
-    GOLDBASE : 1550,
-    PLATINUMBASE : 1700,
-    DIAMONDBASE : 1800,
-    CHAMPIONBASE : 1900,
+  // ELO System Constants
+  DEFAULT_RATING: 1500,
+  DEFAULT_RD: 350,
+  DEFAULT_VOL: 0.06,
```

Comment:
Who was pulling this out? The glicko-2 constants vs the other elo bracket constants.